### PR TITLE
UGDG: Grammar Fix

### PR DESCRIPTION
**Pull request by Berted**
DG:
- Updated acknowledgements

UG:
- Grammar Fix.
- Fix some links, i.e. focus mode command.
- Add details on `g/GRADE_VALUE`
- Add extra note on `a/ADDRESS` constraints.

- Also, fixed FXML versioning issue.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -88,21 +88,21 @@ Note that these commands are also available in focus mode.
...
-Automatically redirects you to this User Guide page.
+Automatically opens your browser and redirects you to this User Guide page.
 
 " %}
 
 
 Format: `help`
-* If the attempt to redirect you to this User Guide page fails, the following help window appears:
-
+* If TA-Assist is unable to redirect you to the page, i.e. due to a non-existant browser, unsupported operating system, etc.,
```
Comment (njxue) [inline docs/UserGuide.md]:
```suggestion
* If TA-Assist is unable to redirect you to the page, i.e. due to a non-existent browser, unsupported operating system, etc.,
```

**Review by njxue — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ If you encounter any issues in launching and using the app, feel free refer to t
 * Items in square brackets are optional.
   * e.g. `n/NAME [c/CLASS_NAME]` can be used as `n/John Doe c/CS1231S` or as `n/John Doe`.
 * Items with `...` after them can be used multiple times.
-  * e.g. `[c/CLASS_NAME...]` can be used as ` ` (i.e. 0 times since it is also optional), `c/CS1101S`, `c/CS2030 c/ST2334` etc.
+  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S`, `c/CS2030 c/ST2334`, etc.
```
Comment (njxue) [inline docs/UserGuide.md:44]:
```suggestion
  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S` or `c/CS2030 c/ST2334`, etc.
```

**Review by njxue — CHANGES_REQUESTED** [review]

**Review by Bubbl3T — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (Berted) [reply docs/UserGuide.md:44]: I think this is okay, since we're have an `etc` at the back?

**Review by njxue — APPROVED** [review]

**Pull request merged**